tools/flye/test-data/result1_scaffolds.fasta

```diff
@@ -1,667 +0,0 @@
->contig_1
```

Comment:
is it correct that this (and the other scaffold files) are now empty?

Reply:
I checked the previous version and they've always been empty. I might need to find other input data in case we want to be sure.

Reply:
Owh no, something changed. The assembly is now full, but the scaffolds stay empty.

Reply:
Flye --> scaffolds.fasta file is a symlink to assembly.fasta, which is retained for the backward compatibility.

In my opinion I can just remove the scaffolds output, because you can link the tools to the new output?